doc/manuals/devel/cookbook.md

```diff
@@ -124,24 +123,23 @@ typedef struct RestService
 } RestService;
 ```
 
-So, to add a REST service eg. `PUT /v2/entities/{EntitId}/attrs/{AttributeName}/metadata/{MetadataName}`, the new item if the RestService vector would look like this:
+So, to add a REST service eg. `PUT /v2/entities/{EntitId}/attrs/{AttributeName}/metadata/{MetadataName}`, the new item if the RestService vector `putServiceV` would look like this:
 
 ```
-{ "PUT", Metadata,  7, { "v2", "entities", "*", "attrs", "*", "metadata", "*" }, "", putMetadata }
+{ Metadata,  7, { "v2", "entities", "*", "attrs", "*", "metadata", "*" }, "", putMetadata }
```

Comment:
Keep an eye on this... as the vector structure change (e.g. to use a continuous string instead of a list of elements) this should evolve.